pages/common/swagger-codegen.md

```diff
@@ -0,0 +1,19 @@
+# swagger-codegen
+
+> Generate code and documentation for your REST api from a OpenAPI/swagger definition
+
+- Generate from a file or an url
+
+`swagger-codegen generate --input-spec {{url_or_file}} ---lang {{language}} --library {{library}}`
+
+- Available languages:
+
+ `swagger-codegen langs`
+
+- Available configuration options
```

Comment:
We should also explain what does `-l java` do in the example. I am guessing it will list configuration options for the java language. We should mention that clearly here.

Also, put java in tokens (`{{}}`)